### ISSUE TITLE
Getting stuck with building win .exe on macos (Also with /PublishReadyToRun false)

### ISSUE THREAD

**Issue by Mondonno** [ElectronNET/Electron.NET#572]
* **Version**:  11.5.1.0

**Node**: 12.14.1
**.NET**: 5.0.203

* **Target**: Windows (the same error when building for Linux) 
*When building for osx it is fine*

Steps to Reproduce:

1. Get on macos laptop
2. Init new project in electron
3. Build it: `electronize /target win /PublishReadyToRun false`
4. Error
*If i do not type /PublishReadyToRun i getting `NETSDK1095` error* #446 #464 

**Comment**

The error content and stack trace:
```
electronize build /target win /PublishReadyToRun false
Build Electron Application...
Build ASP.NET Core App for linux-x64...
Executing dotnet publish in this directory: /Users/x/Documents/Project/obj/desktop/linux
Build ASP.NET Core App for linux-x64 under Release-Configuration...
Microsoft (R) Build Engine 16.9.0+57a23d249 dla platformy .NET
Copyright (C) Microsoft Corporation. Wszelkie prawa zastrzeżone.
  Trwa określanie projektów do przywrócenia...
  Wszystkie projekty są aktualne na potrzeby przywrócenia.
  Project -> /Users/x/Documents/Project/bin/Release/net5.0/linux-x64/Project.dll
  Project -> /Users/x/Documents/Project/bin/Release/net5.0/linux-x64/Project.Views.dll
  Project -> /Users/x/Documents/Project/obj/desktop/linux/bin/
Start npm install...
npm WARN deprecated tslint@6.1.3: TSLint has been deprecated in favor of ESLint. Please see https://github.com/palantir/tslint/issues/4534 for more information.
npm WARN deprecated @types/socket.io-parser@3.0.0: This is a stub types definition. socket.io-parser provides its own type definitions, so you do not need this installed.
npm notice created a lockfile as package-lock.json. You should commit this file.
added 67 packages from 57 contributors and audited 186 packages in 7.232s
found 0 vulnerabilities
ElectronHostHook handling started...
Build Electron Desktop Application...
Executing electron magic in this directory: /Users/x/Documents/Project/bin/desktop
Create electron-builder configuration file...
Package Electron App for Platform linux...
npx: zainstalowano 240 w 24.789s
Cannot find module 'fs/promises'
Require stack:
- /Users/x/.npm/_npx/1236/lib/node_modules/electron-builder/node_modules/builder-util/out/fs.js
- /Users/x/.npm/_npx/1236/lib/node_modules/electron-builder/node_modules/builder-util/out/util.js
- /Users/x/.npm/_npx/1236/lib/node_modules/electron-builder/out/cli/cli.js
- /Users/x/.npm/_npx/1236/lib/node_modules/electron-builder/cli.js
... done
```

also `electronize start` works fine.

**Comment by izakntun** [comment]
I have the same issue last week, the solution was updated NodeJS to the latest version, I used version 10.16 Node and now I have version 14.17 Node.

**Comment by Mondonno** [comment]
@izakntun Ok, after upgrading my node to lastest stable version, i got correct executable. But there is still a bug, you need to put: `/PublishReadyToRun false` in order to build app to any other system than macos.

I think the @GregorBiswanger sholud have a look on this 👍

**Comment by tub5** [comment]
@Mondonno the publish ready issue is already fixed with #570

**Comment by Mondonno** [comment]
@tub5 Thats very good! So it got fixed 1.07.2021 after 17 days of my issue. But still builds does not works on targeted systems

**Comment by GregorBiswanger** [comment]
🎉🚀 New Electron.NET version 23.6.1 released 🚀🎉

With native Electron 23 and .NET 6 support. Your problem should be fixed here. If you continue to have the problem, please let us know. Please note the correct updating of your API & CLI. Info in the README. Have fun!
